web/src/routes/(user)/people/[personId]/+page.svelte

```diff
@@ -288,39 +329,65 @@
-            person={data.person}
-            on:change={(event) => handleNameChange(event.detail)}
-            on:cancel={() => handleCancelEditName()}
-          />
-        {:else}
-          <button on:click={() => (viewMode = ViewMode.VIEW_ASSETS)}>
-            <ImageThumbnail
-              circle
-              shadow
-              url={api.getPeopleThumbnailUrl(data.person.id)}
-              altText={data.person.name}
-              widthStyle="3.375rem"
-              heightStyle="3.375rem"
+      <div role="button" class="relative w-fit p-4 sm:px-6" use:clickOutside on:outclick={() => handleCancelEditName()}>
+        <section class="flex w-96 place-items-center border-black">
+          {#if isEditingName}
+            <EditNameInput
+              person={data.person}
+              suggestedPeople={suggestedPeople.length > 0}
+              bind:name
+              on:change={(event) => handleNameChange(event.detail)}
             />
-          </button>
-
-          <button
-            title="Edit name"
-            class="px-4 text-immich-primary dark:text-immich-dark-primary"
-            on:click={() => (isEditingName = true)}
-          >
-            {#if data.person.name}
-              <p class="py-2 font-medium">{data.person.name}</p>
-            {:else}
-              <p class="w-fit font-medium">Add a name</p>
-              <p class="text-sm text-gray-500 dark:text-immich-gray">Find them fast by name with search</p>
-            {/if}
-          </button>
+          {:else}
+            <button on:click={() => (viewMode = ViewMode.VIEW_ASSETS)}>
+              <ImageThumbnail
+                circle
+                shadow
+                url={api.getPeopleThumbnailUrl(data.person.id)}
+                altText={data.person.name}
+                widthStyle="3.375rem"
+                heightStyle="3.375rem"
+              />
+            </button>
+
+            <button
+              title="Edit name"
+              class="px-4 text-immich-primary dark:text-immich-dark-primary"
+              on:click={() => (isEditingName = true)}
+            >
+              {#if data.person.name}
+                <p class="py-2 font-medium">{data.person.name}</p>
+              {:else}
+                <p class="w-fit font-medium">Add a name</p>
+                <p class="text-sm text-gray-500 dark:text-immich-gray">Find them fast by name with search</p>
+              {/if}
+            </button>
+          {/if}
+        </section>
+        {#if isEditingName}
+          <div class="absolute z-[999] w-96">
+            {#each suggestedPeople as person, index (person.id)}
+              <div
+                class="flex {index === suggestedPeople.length - 1
+                  ? 'rounded-b-lg'
+                  : 'border-b dark:border-immich-dark-gray'} place-items-center bg-gray-100 p-2 dark:bg-gray-700"
+              >
+                <button class="flex w-full place-items-center" on:click={() => suggestPeople(person)}>
```

Comment:
Can you follow the convention of using `handle*` when handling a click event? So `handleSuggestPeople` here.